src/test/suite/stubDemoTutorial.test.ts

```diff
@@ -33,11 +33,29 @@ const TERMINAL_WAIT_RETRY = 2000;
 
 suite('stub out a tutorial', () => {
 
+	let disposables: Disposable[] = [];
+
+	teardown(() => {
+		disposables.forEach(d => d.dispose());
+		disposables.length = 0;
+	});
+
 	test('that we can send an echo command to the terminal and get the response', async () => {
 		const name = 'echoTerminal';
 		const text = `echo \"Hello World ${name}\"`;
 		const result = `Hello World echoTerminal`;
-		await validateTerminalResponse(name, text, result);
+		const winResult = [
+			`Hello`,
+			`World`,
+			`echoTerminal`
+		];
+		await validateTerminalResponseWin(name, text, result);
+
+		// if (process.platform === 'win32') {
+		// 	await validateTerminalResponseWin(name, text, winResult);
+		// } else {
+		// 	await validateTerminalResponse(name, text, result);
+		// }
```

Comment:
what's the link of the snippet updates with this test modification?

Reply:
there is no test at present and never has been a test. if you have an example of how/where to test injecting a snippet, that would be helpful

Reply:
Sorry. this was an errant change for the stub test. Have removed

Reply:
fro testing completion injected: https://github.com/camel-tooling/camel-lsp-client-vscode/blob/eedeccfecdab529721059841367d2ebeb08b1a86/src/ui-test/xml_completion_support_test.ts#L58